app/src/main/java/org/simple/clinic/mobius/MobiusFoo.kt

```diff
@@ -10,3 +10,6 @@ fun <M, F> ((M) -> First<M, F>).toInit(): Init<M, F> =
 
 fun <M, F> justEffect(effect: F): Next<M, F> =
     dispatch<M, F>(setOf(effect))
+
+fun <M, F> next(model: M, effect: F): Next<M, F> =
```

Comment:
Should this take in a `vararg` parameter for effects?

Reply:
Same as above.